### PR TITLE
[#175798637] A user could choose a preferred PSP

### PR DESCRIPTION
**Pull request by vito80ba**
This PR is about a page/function to choose another PSP rather then use the "default" one suggested by PM.

#### List of Changes

- Cosmetic fixes on the page.
- Implementing the call to specific service to obtain the PSP list (To-DO)

#### Motivation and Context

A user, after the input of own credit card, could choose a preferred PSP from a list

#### How Has This Been Tested?
Linter, Puppeteer

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by pasqualespica — APPROVED** [review]
@vito80ba now into `choosepsp` there is call to `getPspListUsingGET` too, but PSP's image logo isn't load with retrieved data

File: src/assets/json/psp.json [hunk 1]
```diff
@@ -1,30 +1,114 @@
 [
-    {
```
Comment (GiovanniMancini) [inline src/assets/json/psp.json:2]: @vito80ba In my opinion, this file shouldn't be stored in assets folder. If it's just a mock we should move it in a dedicated folder

File: src/choosepsp.ts [hunk 2]
```diff
@@ -1,19 +1,74 @@
...
+          )
+          .map(e => ({
+            name: e?.businessName,
+            label: e?.businessName,
+            image: e?.logoPSP,
+            commission: e?.fixedCost?.amount / Math.pow(10, e?.fixedCost?.decimalDigits),
+          })),
+    )
+    .run();
+
```
Comment (GiovanniMancini) [inline src/choosepsp.ts:65]: @vito80ba  @pasqualespica Maybe we should add also the call to `PUT wallet` or do you think we should open another PR?

**Review by GiovanniMancini — CHANGES_REQUESTED** [review]
Maybe we should move the static json in a dedicated folder and add the call to `PUT wallet`

File: src/choosepsp.ts (hunk 2 above)
Comment (pasqualespica) [reply src/choosepsp.ts:63]: @GiovanniMancini we got a discount, for convenience we put everything here
